### PR TITLE
Tune lf model

### PR DESCRIPTION
**Pull request by remi-blandin**

### CODE REVIEW

File: Data.cpp [hunk 1]
```diff
@@ -648,6 +649,7 @@ int Data::synthesizeVowelLf(TlModel *tlModel, LfPulse &lfPulse, int startPos, bo
   Signal singlePulse;
   Signal pulseSignal(BUFFER_LENGTH);
   Signal pressureSignal(BUFFER_LENGTH);
+  Signal noiseSignal(BUFFER_LENGTH);
```
Comment (Simon-Stone) [inline Data.cpp]: This is obsolete, I believe

File: Data.cpp [hunk 2]
```diff
@@ -746,28 +748,28 @@ int Data::synthesizeVowelLf(TlModel *tlModel, LfPulse &lfPulse, int startPos, bo
     // Is a new glottal pulse starting?
     // **************************************************************
 
-    if (i == nextPulsePos)
-    {
-      // Get the pulse amplitude and F0.
+	if (i == nextPulsePos)
```
Comment (Simon-Stone) [inline Data.cpp]: It's annoying but please fix the indentation. The coding style is only two spaces per indentation level.

File: LfPulseDialog.h [hunk 3]
```diff
@@ -54,12 +54,13 @@ class LfPulseDialog : public wxDialog
   // **************************************************************************
 
 private:
-  enum Params
-  {
-    OPEN_QUOTIENT,
-    SHAPE_QUOTIENT,
-    SPECTRAL_TILT,
-    FUNDAMENTAL_FREQUENCY,
+	enum Params
```
Comment (Simon-Stone) [inline LfPulseDialog.h]: Fix Indentation

**Review by Simon-Stone — CHANGES_REQUESTED** [review]
Please fix the indentation and please delete the sig.txt file from the repo.

File: Data.cpp (hunk 1 above)
Comment (remi-blandin) [reply Data.cpp]: Yes indeed

File: Data.cpp (hunk 2 above)
Comment (remi-blandin) [reply Data.cpp]: Ok, no pb :)

**Review by Simon-Stone — APPROVED** [review]

**Pull request merged**